src/app/views/layouts/_navigation.html.erb

```diff
@@ -1,6 +1,6 @@
 <ul class="nav navbar-nav">
-  <li><%= link_to 'Current Quizzes', '#' %></li>
-  <li><%= link_to 'Completed Quizzes', '#' %></li>
+  <li><%= link_to 'Current Quizzes', quizzes_path %></li>
+  <li><%= link_to 'Completed Quizzes', quizzes_path %></li>
```

Comment:
I think these are the wrong paths because neither of them will list what the name says.